src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -2070,12 +2066,9 @@ private void doDeleteRoute(UUID guid) {
                 .block();
     }
 
-    private void doDeleteServiceInstance(CloudServiceInstance serviceInstance) {
-        UUID serviceInstanceGuid = serviceInstance.getMetadata()
-                                                  .getGuid();
-        delegate.serviceInstances()
+    private void doDeleteServiceInstance(UUID serviceInstanceGuid) {
+        delegate.serviceInstancesV3()
                 .delete(DeleteServiceInstanceRequest.builder()
-                                                    .acceptsIncomplete(true)
                                                     .serviceInstanceId(serviceInstanceGuid.toString())
                                                     .build())
                 .block();
```

Comment:
Deleting a managed service instance returns a job ID. It should be handled

Reply:
Again last operation used here